packages/core/src/routes/admin-user.test.ts

```diff
@@ -308,11 +308,27 @@ describe('adminUserRoutes', () => {
       }
     });
     await expect(
-      userRequest.patch(`/users/${notExistedUserId}/roleNames`).send({ roleNames: ['admin'] })
```

Comment:
do we still have the roleNames path?

Reply:
we have. refer to https://github.com/logto-io/logto/blob/b6ae03e1dcba0421fc21cd43850c5b710c230085/packages/core/src/routes/admin-user.ts#L183